development/compiling/compiling_for_linuxbsd.rst

```diff
@@ -55,8 +55,8 @@ Distro-specific one-liners
 +------------------+-----------------------------------------------------------------------------------------------------------+
 | **FreeBSD**      | ::                                                                                                        |
 |                  |                                                                                                           |
-|                  |     sudo pkg install scons pkgconf xorg-libraries libXcursor libXrandr libXi xorgproto libGLU alsa-lib \  |
-|                  |         pulseaudio yasm                                                                                   |
+|                  |     sudo pkg install py37-scons pkgconf xorg-libraries libXcursor libXrandr libXi xorgproto libGLU \      |
```

Comment:
Is there no alias that would be more version agnostic? This sounds like it will break if/when freebsd updates it's py3 version to 3.8, 3.9 or 3.10.